### ISSUE TITLE
Updates to link tags using rel="preload" can revert onload behavior

### ISSUE THREAD

**Issue by brendean** [nfl/react-helmet#170]
When using server side rendering if you are using a `<link>` with `rel="preload"` ([more info on preload](https://www.w3.org/TR/preload/)) and you change the `rel` after the resource loads (via the `onload` event handler) you don't want `react-helmet` to update the tags back to `rel="preload"` instead you would want any `<link rel="preload">` tag to not be updated.

The simplest scenario to show this is:
1. the server sends HTML with `<link rel="preload">`
2. the `<link>` resource loads and the `onload` event handler updates `rel` to `stylesheet`
3. Javascript bundle loads
4. react app renders and `react-helmet` reverts back to `<link rel="preload">`

**Comment by potench** [comment]
Can you control this attribute with `canUseDOM`? https://github.com/JedWatson/exenv

```
import {canUseDOM} from "exenv";
//...

link={[
   {"rel":  canUseDOM ? "stylesheet" : "preload", "href": "some.css"},
```

**Comment by potench** [comment]
Hmm I guess you'd still need to wait for the `onLoad` callback before switching to `rel=stylesheet`

**Comment by cwelch5** [comment]
Yeah, you don't know whether the bundle loads or the link finishes preloading first.  We'd need to handle the flipping of `preload` to `stylesheet` in Helmet.  This is a very special case, might be easier to manage this case out of Helmet.
